source/usage-examples.txt

```diff
@@ -48,22 +48,49 @@ or you can
 Once you have imported the dataset, you can copy and paste a usage
 example into your development environment of choice. You can follow the
 :doc:`quick start guide </quick-start>` to learn more about getting
-started with the MongoDB Go Driver. Once you've copied a usage example,
-you'll need to edit the connection string to get the example connected to
-your instance of MongoDB:
+started with the MongoDB Go Driver. 
+
+Once you've copied a usage example, you'll need to edit and place
+:ref:`your connection string <usage-examples-connection-string>` in
+:ref:`your environment variable <usage-examples-env-variable>` to get
+the example connected to your instance of MongoDB. 
 
 .. code-block:: go
+   
+   uri := os.Getenv("MONGODB_URI")
+
+.. _usage-examples-env-variable:
+
+Environment Variable
+~~~~~~~~~~~~~~~~~~~~
+
+You can use `GoDotEnv <https://github.com/joho/godotenv>`__ to set up
+your environment variable. 
 
-   // Replace the following with your MongoDB deployment's connection string.
-   uri := "mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority";
+When you get to the "Usage" section, add the following application
+configuration in your ``.env`` file at the root of your project: 
```

Comment:
Change: At first I didn't know which Usage section you were talking about. I think I would include this as additional context instead.

```suggestion
Add the following application configuration in your ``.env`` 
file at the root of your project. For more information, see the
`GoDotEnv documentation <https://github.com/joho/godotenv#usage>`__.
```